### ISSUE TITLE
Can VLC be used on raspberries?

### ISSUE THREAD

**Issue by bbhxwl** [picoe/Eto#1990]
Can VLC be used on raspberries?
At present, we know a lot of cross platform methods, which are difficult to implement.

**Comment by cwensley** [comment]
If you can get VLC working on a [GtkSharp](https://github.com/GtkSharp/GtkSharp) based application, then yes you could also do it with Eto.Forms.

**Comment by bbhxwl** [comment]
> If you can get VLC working on a [GtkSharp](https://github.com/GtkSharp/GtkSharp) based application, then yes you could also do it with Eto.Forms.

I haven't run GTKSharp successfully on raspberry pi, only on windows. If I can't run it on raspberry pie, I can use WinForm directly on windows. Are there any examples?

**Comment by cwensley** [comment]
No, there are no examples of using VLC in Eto.  There may be a ton for WinForms, Wpf, Gtk, Mac, etc, but I haven't looked.

**Comment by bbhxwl** [comment]
Well, thank you. At present, no instance can run VLC on raspberry

**Comment by cwensley** [comment]
Note that to use it one would only have to implement a [Custom Platform Control](https://github.com/picoe/Eto/wiki/Custom-Platform-Controls) for each platform you wish to use.

**Comment by cwensley** [comment]
[LibVLCSharp](https://code.videolan.org/videolan/LibVLCSharp) looks like a promising way to use it with most of the platforms Eto supports.  The GTK implementation is still stuck at Gtk#2 though 😞

**Comment by bbhxwl** [comment]
> [LibVLCSharp](https://code.videolan.org/videolan/LibVLCSharp) looks like a promising way to use it with most of the platforms Eto supports. The GTK implementation is still stuck at Gtk#2 though 😞

I hope ETO can support cross platform VLC in the future
